src/reference/asciidoc/ip.adoc

```diff
@@ -364,6 +364,41 @@ Initially, the first factory in the configured list is used; if a connection sub
 
 NOTE: When using the failover connection factory, the singleUse property must be consistent between the factory itself and the list of factories it is configured to use.
 
+[[affinity-cf]]
```

Comment:
`tcp-affinity-cf` then?

Who knows what else we will introduce in the future...